src/backend/utils/logger.js

```diff
@@ -5,13 +5,24 @@ const os = require('os');
 require('../lib/config');
 
 let logger;
+let lLog;
 
 /* If in Development mode:
  * Output logs to console (which is by default)
  * Enable prettyPrint option and translate time from epoch time to local time.
  * Set log level to LOG_LEVEL environment variable with 'debug' as default level.
  */
-if (process.env.NODE_ENV === 'development') {
+
+/* Created a local variable to to hold the value form process.env.LOG_LEVEL.toLowerCase()
+ * The if statement will check the verible if the verible is anything other than levels
+ * stated within file it will automaticaly set to info.
+ * https://getpino.io/#/docs/api?id=level-string Look for the title ``` level (String) ```
+ */
```

Comment:
We need to be careful here.  You've got a good start at dealing with issues but there are more cases we have to worry about.  For example, we care about `process.env.LOG_LEVEL` being undefined, non-lowercase (both of which you have), and also whether it is one of the default log levels (trace, info, debug, etc).

To check the latter, you can so something like this:

```js
logLevel = (process.env.LOG_LEVEL || 'info').toLowerCase();
// Make sure logLevel is one of the default pino log levels (debug, trace, error, info, etc)
if(!pino.levels.values[logLevel]) {
  logLevel = 'info';
}
```

After this, `logLevel` should be safe to use, and should be `info` or one of the other levels.